energywebfoundation/ocn-tools-elia
# Feat/asset dids

Added an initial implementation for asset DID creation, whereby the operator is set to the asset's real-life operator (MSP or CPO). 

Also wired up automatic creation of asset DIDs on startup (behind a feature flag). 

To test:
- Set `createAssetDIDs` to `true` in `src/config.ts`
- Make sure to fund the key in the `OCN_IDENTITY` environment variable with volta tokens
